docs/docs-beta/docs/guides/monitor-alert/custom-logging.md

```diff
@@ -3,4 +3,96 @@ title: "Setting up custom logging"
 sidebar_position: 20
 ---
 
-# Setting up custom logging
+
+# Custom loggers
+
+Customize loggers to emit your logs in a way that's more useful to you, whether to make them more readable or
+easier to process by your log management system.
+
+
+## What you'll learn
+
+- How to write a custom logger
+- How to configure your jobs to use this custom logger
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- A basic understanding of Dagster concepts such as assets, jobs and definitions
+- A working knowledge of the Python logging module
+
+</details>
+
+
+## Step 1: Add a prebuilt custom logger to your jobs
+
+This step shows how to add an existing custom logger to your jobs.
+
+### Add the custom logger to your asset jobs
+
+The following example shows how to add the custom logger to your code location definitions and configure an asset job
+to use it.
+
+<CodeExample filePath="guides/monitor-alert/custom-logging/asset-job-example.py" language="python" title="Add custom logger to asset job" />
+
```

Comment:
Consider showing example output of the `json_console_logger`.

Reply:
added